src/components/organisms/CourseDashboard.tsx

```diff
@@ -29,6 +45,66 @@ const CourseDashboard = ({ term }: { term: Term }) => {
           </Text>
         </HStack>
       </HStack>
+      <Center>
+        <VStack
+          p={2}
+          margin={3}
+          border="solid"
+          borderRadius={10}
+          borderWidth={1}
+        >
+          <Text
+            textAlign="center"
+            fontWeight="medium"
+            fontSize="xl"
+            letterSpacing={3}
+            textTransform="uppercase"
+          >
+            Frequent Actions
```

Comment:
I think all these frequent actions should be added to the sample data and rendered by the UI, not hard coded by the UI directly. Look at how I implemented everything for the professors using the types for colors/professors, Rebecca also added discussion posts to it.

Other than that I just fixed some UI stuff, but feel free to merge whenever you think it's good @Shake1999. If you do have the time to put the data in `sampleData` that'd be ideal